build/exhibitor-tls-artifacts.tpl

```diff
@@ -1,5 +1,31 @@
 #!/usr/bin/env bash
 
-# set -ex
+set -e
 
-docker run -it --rm -v $(pwd):/build --workdir /build {{DOCKER_IMAGE}} $@
+PARAMS=""
+OUTPUT_DIRECTORY="$(pwd)"
+
+while (( "$#" )); do
```

Comment:
Let's please not do this and instead use Python for CLIs - happy to discuss!

Reply:
@mhrabovcin Thoughts? @adamtheturtle, we decided to use a shell script and docker to remove the python3.5+, openssl-dev, and java dependencies on the user environment.